core/src/main/java/com/homeaway/streamplatform/streamregistry/db/dao/AbstractDao.java

```diff
@@ -157,10 +156,13 @@ public void updateAvroStream(AvroStream stream) {
             actorBuilder.setRegionStreamConfigurations(new ArrayList<>());
             Map<String, String> configMap = new HashMap<>();
 
```

Comment:
correct 👍 
the Topic configuration has nothing to do with Producer/Consumer configurations